packages/vendure-plugin-goedgepickt/src/api/goedgepickt.service.ts

```diff
@@ -389,15 +391,21 @@ export class GoedgepicktService
       if (order.state === 'Delivered') {
         return;
       }
-      await transitionToDelivered(this.orderService, ctx, order, {
-        code: goedgepicktHandler.code,
-        arguments: [
-          {
-            name: 'goedGepicktOrderUUID',
-            value: orderUuid,
-          },
-        ],
-      });
+      //FIX ME
```

Comment:
This is probably not needed anymore if you merge `v2` into the current branch, because test/util have been merged